docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1568 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    gd_series = cudf.from_pandas(pd_series)
</pre><p>Combine so all this is in one cell:</p><p> </p><p><code>num_rows = 300_000_000</code></p><p><code>pd_series = ...</code></p><p><code>gd_series = ...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>